### PR TITLE
[OCPQE-13828] - Reserve Bare Metal servers by Vendor

### PR DESCRIPTION
**Pull request by bmanzari**

### CODE REVIEW

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-baremetal-ephe-sno-agent-ipv4-static-connected-p1-f14

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-baremetal-ephe-sno-agent-ipv4-static-connected-p1-f14

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-baremetal-ephe-sno-agent-ipv4-static-connected-p1-f14

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-sno-agent-ipv4-static-connected-f7

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-sno-agent-ipv4-static-connected-f7

File: ci-operator/step-registry/baremetal/lab/agent/install/network/static/ephemeral/baremetal-lab-agent-install-network-static-ephemeral-workflow.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 workflow:
   as: baremetal-lab-agent-install-network-static-ephemeral
   steps:
+    env:
+      VENDOR: dell
```
Comment (aleskandro) [inline ci-operator/step-registry/baremetal/lab/agent/install/network/static/ephemeral/baremetal-lab-agent-install-network-static-ephemeral-workflow.yaml:5]: Mmm, I wouldn't do this here as it's a bit hidden and will require a different setup with the explicit vendor only for the arm64 jobs (hpe)

Comment (bmanzari) [reply ci-operator/step-registry/baremetal/lab/agent/install/network/static/ephemeral/baremetal-lab-agent-install-network-static-ephemeral-workflow.yaml]: Moved to https://github.com/openshift/release/pull/46996/files#diff-50397b8e1c037c03c575bb66443685d531f5333a1885375f94d7538e62e82591R1918 and

**Comment by aleskandro** [comment]
/lgtm
/approve

(you'd remove the wip)

**Comment by bmanzari** [comment]
/override ci/rehearse/periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-sno-agent-ipv4-static-connected-f7

**Comment by aleskandro** [comment]
Hi Biagio, I think you mostly to rebase this PR for making it land.

**Pull request merged**